__tests__/api/profiles/get-other-profile-by-username.test.js

```diff
@@ -0,0 +1,54 @@
+const request = require('supertest');
+const app = require('../../../app');
+const generateUserAndFollowSeeds = require('../__utils__/seeds/users_and_follow_seeds');
+
+beforeEach(async () => {
+  await generateUserAndFollowSeeds();
+});
+
+const arrayOfDetailUser = expect.objectContaining({
+  user_id: expect.any(String),
+  country_code: expect.any(String),
+  username: expect.any(String),
+  real_name: expect.any(String),
+  last_active_at: expect.any(String),
+  profile_pic_path: expect.any(String),
+  profile_pic_asset_id: expect.any(String),
+  profile_pic_public_id: expect.any(String),
+  status: expect.any(String),
+  bio: expect.any(String),
+  is_banned: expect.any(Boolean),
+  is_following: expect.any(Boolean),
+  is_anonymous: expect.any(Boolean),
+  allow_anon_dm: expect.any(Boolean),
+  isSignedMessageEnabled: expect.any(Boolean),
+  isAnonMessageEnabled: expect.any(Boolean),
+  only_received_dm_from_user_following: expect.any(Boolean),
+  following_symbol: expect.any(String),
+  follower_symbol: expect.any(String),
+  verified_status: expect.any(String),
+  created_at: expect.any(String),
+  updated_at: expect.any(String)
+});
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

While this test setup is generally fine, it's worth noting that the `arrayOfDetailUser` object is quite large and may be difficult to maintain if the user model changes frequently. Consider breaking it down into smaller chunks or using a factory function to generate expected objects.



<!-- This is an auto-generated comment by OSS CodeRabbit -->